mobile/src/bundles/common/components/button/styles.ts

```diff
@@ -1,13 +1,89 @@
 import { StyleSheet } from 'react-native';
 
 const styles = StyleSheet.create({
```

Comment:
There are a lot of repeatings in styles, think of optimising them after adding button type and refactoring the pressed state

Reply:
fix